<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VulnusEditorClass</class>
 <widget class="QMainWindow" name="VulnusEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>692</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vulnus - (c)2015 Krien Linnenbank</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="woundTimelineBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>641</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Wound timeline</string>
    </property>
    <widget class="QSlider" name="timelineSlider">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>621</width>
       <height>19</height>
      </rect>
     </property>
     <property name="maximum">
      <number>199</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="invertedAppearance">
      <bool>false</bool>
     </property>
     <property name="invertedControls">
      <bool>false</bool>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
     <property name="tickInterval">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="cacheProgressBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>621</width>
       <height>11</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="woundTimelineLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>621</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="VulnusEditor.qrc">:/VulnusEditor/Wound_healing_phases.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="mainViewLabel" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionOpen_2"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPaintWound">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Paint wound</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionOpen_2">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="VulnusEditor.qrc"/>
 </resources>
 <connections/>
</ui>
